rpd_generator/bdl_structure/bdl_commands/zone.py

```diff
@@ -321,6 +327,58 @@ def populate_data_elements(self):
                     zone_ef_power_per_flow * exhaust_airflow
                 )
             return
+        # Populate Terminal for terminals_has_demand_control_ventilation
+        if self.keyword_value_pairs.get(BDL_ZoneKeywords.OA_FLOW_PER) is not None:
+            if (
+                self.parent.keyword_value_pairs.get(BDL_SystemKeywords.DOA_SYSTEM)
+                is not None
+            ):
+
+                if (
+                    self.parent.keyword_value_pairs.get(
+                        BDL_SystemKeywords.MIN_OA_METHOD
+                    )
+                    == BDL_SystemMinimumOutdoorAirControlOptions.DCV_ZONE_SENSORS
```

Comment:
I am curious what happens if DOAS_ATTACHED_TO=CONDITIONED_ZONES & MIN_OA_METHOD = DCV_RETURN_SENSOR

Help text says `If a DOAS supplies ventilation air directly to the zones, this mode is not supported. Instead, demand-controlled ventilation using the DCV-ZONE-SENSORS mode is assumed.` Not clear what the BDL looks like when this combination of options is selected, and not clear how the DCV behaves if it is used.